app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -186,6 +187,10 @@ public function panel(Panel $panel): Panel
                     ->label('Profile Settings')
                     ->url(fn () => EditProfile::getUrl())
                     ->icon('heroicon-s-cog-6-tooth'),
+                MenuItem::make()
+                    ->label('About')
+                    ->url("javascript:window.dispatchEvent(new CustomEvent('open-modal',{detail:{id:'about'}}))")
```

Comment:
To match the other suggestion.

```suggestion
                    ->url("javascript:window.dispatchEvent(new CustomEvent('open-modal', {detail:{id:'app-about'}}))")
```